README.md

```diff
@@ -21,9 +21,9 @@ from pyDownload import Downloader
 
 url = 'https://github.com/party98/Python-Parallel-Downloader/archive/master.zip'
 downloader = Downloader(url=url, filename=filename)
-if downloader.is_running():
+if downloader.is_running:
     time.sleep(1)
-print('File downloaded to %s' % downloader.get_file_name())
+print('File downloaded to %s' % downloader.get_file_name)
```

Comment:
the property name has been changed to `file_name` [here](https://github.com/party98/pyDownload/blob/c4b10f3dec27934f05fff702783ee74dd7cc08fc/pyDownload/downloader.py#L19)